src/custom-gpt-evaluator.js

```
@@ -0,0 +1,215 @@
+  } catch (error) {
+    console.error('Custom GPT evaluation failed:', error);
+    return {
+      score: 0,
+      decision: 'ERROR',
+      error: error.message,
+      processingTime: Date.now() - startTime
+    };
+  }
+}
+
+async function evaluateInstructionAdherence(prompt, response, expectedFormat) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'evaluateInstructionAdherence'. Consider to delete it or rename it.

'evaluateInstructionAdherence' is defined here:


(lint/suspicious/noRedeclare)